Specs/Core/EllipsoidSpec.js

```diff
@@ -3,6 +3,7 @@ import { Cartographic } from "../../Source/Cesium.js";
 import { Ellipsoid } from "../../Source/Cesium.js";
 import { Math as CesiumMath } from "../../Source/Cesium.js";
 import createPackableSpecs from "../createPackableSpecs.js";
+import { Rectangle } from "../../Source/Cesium.js";
```

Comment:
Just to keep things tidy, move this `import` above one line so it's next to the other `Cesium.js` imports.